make/Main.gmk

```diff
@@ -1334,6 +1340,7 @@ RUN_TEST_TARGETS := $(addprefix run-test-, $(ALL_NAMED_TESTS))
 
 run-test: test
 exploded-run-test: exploded-test
+run-test-docs: test-docs
```

Comment:
`run-tests` is legacy support for a deprecated syntax. It should not be extended to new targets.